deploy/examples/operator.yaml

```diff
@@ -514,6 +514,8 @@ data:
 
   # Enable topology based provisioning.
   CSI_ENABLE_TOPOLOGY: "false"
+  # RevisionHistoryLimit value for all deployments created by rook.
+  ROOK_REVISION_HISTORY_LIMIT: "3"
```

Comment:
Let's comment this out for users so we can keep current default behavior for all users who may be relying on it.

Reply:
done